.github/workflows/test_action.yml

```diff
@@ -0,0 +1,17 @@
+on:
+  pull_request:
```

Comment:
Pull requests from a fork will have READ permissions only and thus can not post a comment on the pull request itself. One can use `pull_request_target`, which will grant the run WRITE permissions. That said, this needs to be used with care, e.g. by tightly scoping the additional permissions to trusted logic only.

See https://stackoverflow.com/questions/69499645/how-to-securely-allow-github-actions-to-check-pr-and-post-results-in-comment for details.